assets/js/modules/analytics/common/create-account-field.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * CreateAccountField component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	Input,
+	TextField,
+} from '../../../material-components';
+import classnames from 'classnames';
+
+const CreateAccountField = ( {
+	validationIssues,
+	setValidationIssues,
+	objectValue,
+	objectSetter,
+	name,
+	label,
+} ) => {
+	return (
+		<TextField
+			className={ classnames(
+				'mdc-text-field',
+				{ 'mdc-text-field--error': validationIssues[ name ] }
+			) }
+			label={ label }
+			name={ name }
+			onChange={ ( e ) => {
+				validationIssues[ name ] = '' === e.target.value;
```

Comment:
This is mutating the prop which could cause problems if that in turn changes the value outside of the component by reference.

Better to just merge it in before/while setting:
```js
setValidationIssues( {
    ... validationIssues,
    [ name ] : '' === e.target.value,
} );
```

Reply:
Great suggestion, thank you!

Reply:
Updated in https://github.com/google/site-kit-wp/pull/1389/commits/d445f69708dc5f8634749d78613eb4680eb3b509.